src/components/SummaryWeather.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useWeather } from './WeatherProvider';
+import {
+  toCapitalized,
+  toLocalDate,
+  getHourString,
+  getIconSrc,
+} from '../utilities/helperFunctions';
+
+export default function SummaryWeather() {
+  const { currentWeather, hourlyWeather } = useWeather();
+  if (!(currentWeather && hourlyWeather)) return null;
+
+  const { timezone } = currentWeather;
+  const summaryHeader = `${currentWeather.name}, ${currentWeather.sys.country}`;
+  const summaryIconURL = getIconSrc(currentWeather.weather[0].icon);
+  const summaryIconAltText = `An icon representing ${currentWeather.weather[0].description}`;
+  const summaryDescription = `${toCapitalized(
+    currentWeather.weather[0].description
+  )}`;
+  const summaryTemperature = `${Math.round(currentWeather.main.temp)}°`;
+
+  const hourlyForecast = hourlyWeather.list
+    .slice(0, 4)
+    .map((weatherForHour) => (
+      <div key={weatherForHour.dt} className="row align-items-center">
+        <div className="col" />
```

Comment:
I don't think this div is necessary because in the next line, you specify a div with a col attribute as well. With Bootstrap grid, you only need a parent row with a single col child. I'm assuming you're approach to display the hourly weather is to wrap each hour in a row with only one column right?